src/auth/auth.module.ts

```diff
@@ -10,7 +9,6 @@ import { AuthController } from './auth.controller';
         JwtModule.registerAsync({
             useClass: ConfigService,
         }),
-        UserModule,
```

Comment:
Why did u remove this UserModule which the exact same thing you removed from AdminSerivce

Reply:
Sorry I have just seen `@Global` in user module file.